src/main/java/com/solvd/android/HomePage.java

```diff
@@ -0,0 +1,152 @@
+package com.solvd.android;
+
+import com.zebrunner.carina.utils.mobile.IMobileUtils;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.security.cert.X509Certificate;
+import java.util.List;
+import java.util.Optional;
+
+public class HomePage extends AbstractPage implements IMobileUtils {
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/iv_task_add')]")
+    private ExtendedWebElement addTaskButton;
+
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/task_create_input')]")
+    private ExtendedWebElement inputTextBar;
+
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/task_create_btn')]")
+    private ExtendedWebElement submitButton;
+
+    @FindBy(xpath = "//*[contains(@text, '%s')]")
+    private ExtendedWebElement task;
+
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/task_detail_more')]")
+    private ExtendedWebElement taskDetailsButton;
+
+    @FindBy(xpath = "//*[contains(@resource-id, 'd/detail_delete')]")
+    private ExtendedWebElement taskDetailsDeleteButton;
+
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/dialog_confirm')]")
+    private ExtendedWebElement confirmDeleteTaskButton;
+
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/tag_management')]")
+    private ExtendedWebElement detailsButton;
+
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/popup_tv') and @text=\"Manage Categories\"]")
+    private ExtendedWebElement manageCategoriesButton;
+
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/create_category_text')]")
+    private ExtendedWebElement createNewCategoryButton;
+
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/dialog_input')]")
+    private ExtendedWebElement input;
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/dialog_confirm')]")
+    private ExtendedWebElement saveCategoryButton;
+
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/toolbar_back')]")
+    private ExtendedWebElement backToHomePageButton;
+
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/category_text') and @text=\"Wishlist\"]")
+    private ExtendedWebElement wishListCategory;
+
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/categorymag_layout')]/android.view.ViewGroup")
+    private List<Category> categories;
+
+    @FindBy(xpath = "//*[contains(@resource-id, 'id/dialog_confirm')]")
+    private ExtendedWebElement confirmDeleteCategory;
+
+    @FindBy(xpath = ".//*[@text=\"Delete\"]")
+    private ExtendedWebElement deleteCategoryButton;
```

Comment:
all actions with categories are happening on the separate page (Manage Categories)